test/unit/Cardano/Wallet/DB/MVarSpec.hs

```diff
@@ -227,8 +228,8 @@ deriving instance Show (PrimaryKey WalletId)
 instance Arbitrary (PrimaryKey WalletId) where
     shrink _ = []
     arbitrary = do
-        k <- choose (0, 10)
-        return $ PrimaryKey $ WalletId $ UUID.fromWords k 0 0 0
+        bytes <- B8.pack . pure <$> elements ['a'..'k']
+        return $ PrimaryKey $ WalletId $ hash bytes
```

Comment:
So here, we try to keep the generator's entropy quite low because we do want to generate conflicts in the properties.